MAUI/Autocomplete/UI-Customization.md

```diff
@@ -11,6 +11,10 @@ documentation: ug
 
 This section explains different UI customizations available in the [.NET MAUI Autocomplete](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.SfAutocomplete.html) control.
 
+Watch this video to quickly customize the .NET MAUI Autocomplete.
```

Comment:
Watch this video for a quick guide on customizing the .NET MAUI Autocomplete.

Reply:
Content modified